### ISSUE TITLE
milliForth needs proof

### ISSUE THREAD

**Issue by jart** [fuzzballcat/milliForth#8]
Hello. You know me as the author of SectorLISP and SectorLambda. You have a cool project. Congratulations on your claim to have developed the world's tiniest programming language. When I published SectorLISP, I included both an implementation of a Turing machine, as well as an example of how SectorLISP can be used to implement itself. Could you do that too?

**Comment by fuzzballcat** [comment]
Yes, definitely!  I'll have to think about the second one (to be honest I'm not entirely sure what the canonical metacircular evaluator is for FORTH), but when I get time this is top of my list currently.

**Comment by ingenieroariel** [comment]
@fuzzballcat if you use Discord it would be great to have you at the Cosmopolitan Discord Server where a few people are trying to understand how milliForth works.

For example
```forth
\\ PRIMITIVES \\
\ Duplicate the top of the stack by getting its memory address and reading it
: dup sp@ @ ;

\\ NUMBER LITERALS \\
\ -1 constant.
\ s@ is a pointer to the state struct at 0x1000
\ 0x1000 NAND 0x1000 = 0xEFFF
\ 0xEFFF NAND 0x1000 = 0xFFFF
: -1 s@ s@ nand s@ nand ;
\ 0xFFFF NAND 0xFFFF = 0x0000
: 0 -1 dup nand ;
\ 0xFFFF + 0xFFFF = 0xFFFE (0x1FFFE)
\ 0xFFFE NAND 0xFFFE = 0x0001
: 1 -1 dup + dup nand ;
\ Creating further number literals is trivial
: 2 1 1 + ;
: 4 2 2 + ;
: 6 2 4 + ;
```

https://discord.gg/8t92w3YCsg

**Comment by fuzzballcat** [comment]
As you might have noticed, there's now a `bf.FORTH` file which implements a little brainf**k interpreter to show turing completeness!
